internal/app/exporter/exporter.go

```diff
@@ -407,6 +409,28 @@ func (c *rdsCollector) getQuotasMetrics(client servicequotas.ServiceQuotasClient
 	c.logger.Debug("quota metrics fetched", "metrics", metrics)
 }
 
+func (c *rdsCollector) getInstanceTagLabels(dbidentifier string, instance rds.RdsInstanceMetrics) (keys []string, values []string) {
+	labels := map[string]string{
+		"aws_account_id": c.awsAccountID,
+		"aws_region":     c.awsRegion,
+		"dbidentifier":   dbidentifier,
+	}
+
+	// Add instance tags to labels
+	// Prefix label containing instance's tags with "tag_" prefix to avoid conflict with other labels
+	for k, v := range instance.Tags {
+		labelName := fmt.Sprintf("tag_%s", k)
+		labels[labelName] = v
+	}
+
+	for k, v := range labels {
+		keys = append(keys, k)
+		values = append(values, v)
+	}
+
+	return
```

Comment:
```suggestion
	return keys, values
```

I know this is a feature of Go to be able to return values if you named the return values in the function signature, but I find it harder to read and would advise against it unless the function is < 5 lines.

Reply:
Agree, it's more visible. I applied it.